### DISCUSSION TITLE
Clone table fields missing

### DISCUSSION THREAD

**Discussion by Januaryyiyue** [milaboratory/mixcr#1572] [Q&A] [answered]
## Checklist before submitting the issue:

 - [X] The issue is strongly related to the MiXCR software
 - [ ] The issue can be reproduced with [the most recent version](https://github.com/milaboratory/mixcr/releases) of MiXCR
 - [X] There is no answer to the question in the [official documentation](http://mixcr.readthedocs.io/) and there is no duplicate issue in the [bug tracker](https://github.com/milaboratory/mixcr/issues?utf8=✓&q=)  
 - [ ] Inspection of raw alignments with [exportAlignmentsPretty](http://mixcr.readthedocs.io/en/master/export.html#exporting-well-formatted-alignments-for-manual-inspection) shows that data has the expected architecture, and sample preparation artefacts are not the reason of the problem (if this is the matter of the issue)

## Expected Result

A clone table with the following fields: `cloneId	cloneCount	cloneFraction	targetSequences	targetQualities	allVHitsWithScore	allDHitsWithScore	allJHitsWithScore	allCHitsWithScore	allVAlignments	allDAlignments	allJAlignments	allCAlignments	nSeqFR1	minQualFR1	nSeqCDR1	minQualCDR1	nSeqFR2	minQualFR2	nSeqCDR2	minQualCDR2	nSeqFR3	minQualFR3	nSeqCDR3	minQualCDR3	nSeqFR4	minQualFR4	aaSeqFR1	aaSeqCDR1	aaSeqFR2	aaSeqCDR2	aaSeqFR3	aaSeqCDR3	aaSeqFR4	refPoints`

## Actual Result

Fewer fields than expected" `cloneId	readCount	readFraction	targetSequences	targetQualities	allVHitsWithScore	allDHitsWithScore	allJHitsWithScore	allCHitsWithScore	allVAlignments	allDAlignments	allJAlignments	allCAlignments	nSeqCDR3	minQualCDR3	aaSeqCDR3	refPoints`

## Exact MiXCR commands 

`java -jar /path/to/mixcr.jar align -p rnaseq-cdr3 --verbose -s hsa -OallowPartialAlignments=true -OvParameters.geneFeatureToAlign=VGeneWithP -r /path/to/LOG_ALIGN.txt /path/to/file_R1.fastq /path/to/gile_R2.fastq /path/to/alignments_file.vdjca
java -jar /path/to/mixcr.jar assemblePartial /path/to/alignments_file.vdjca /path/to/rescue1_file.vdjca
java -jar /path/to/mixcr.jar assemblePartial /path/to/rescue1_file.vdjca /path/to/rescue2_file.vdjca
java -jar /path/to/mixcr.jar extend /path/to/rescue2_file.vdjca /path/to/extended_file.vdjca
java -jar /path/to/mixcr.jar assemble -r /path/to/LOG_ASSEMBLE_file.txt /path/to/extended_file.vdjca /path/to/file.clns
java -jar /path/to/mixcr.jar exportClones --chains TRB /path/to/file.clns /path/to/CLONES_TRB_file.tsv
java -jar /path/to/mixcr.jar exportClones -o -t --chains TRB /path/to/file.clns /path/to/PROD_CLONES_TRB_file.tsv`

**Comment by mizraelson** [comment, marked as answer]
Hi, I recommend running the latest version of MiXCR and the following command:

```
java -jar /path/to/mixcr.jar analyze rna-seq \
-s hsa \
/path/to/file_R1.fastq \
/path/to/gile_R2.fastq \
/path/to/output
```

It will run all the commands you have, and also `assembleContigs` that seems to be missing from your pipeline. The output should be as you expected with all regions if covered.

**Reply by Januaryyiyue** [reply]
Thank you for the suggestion. I've switched to the newest version of MiXCR with a custom preset inherited from `generic-amplicon-with-umi`.

The start of the yaml file goes like this:
```
inheritFrom: generic-amplicon-with-umi
pipeline:
  - align
  - refineTagsAndSort
  - assemble
  - exportClones
```
 The end results still only has the fields:
`cloneId	readCount	readFraction	targetSequences	targetQualities	allVHitsWithScore	allDHitsWithScore	allJHitsWithScore	allCHitsWithScore	allVAlignments	allDAlignments	allJAlignments	allCAlignments	nSeqCDR3	minQualCDR3	aaSeqCDR3	refPoints
`
Now I am not sure what step(s) are missing. Thank you so much!

**Reply by mizraelson** [reply]
Do you have a bulk RNA-seq data?

**Reply by Januaryyiyue** [reply]
We have hybrid capture DNA-sequencing data of the CDR3 region.

Should I try adding `- assembleContigs` in the yaml file?

**Reply by mizraelson** [reply]
Are you talking about CapIG-seq? The library prepared by this protocol does not cover other regions as I understand and primarily is focused on CDR3.

**Reply by Januaryyiyue** [reply]
I am using capTCR-seq, which only sequences the CDR3 region.

**Reply by mizraelson** [reply]
Then you can't export columns with sequences of the regions that are not covered.
